Automattic/pocket-casts-android
# Onboarding: Add welcome confetti

| 📘 Project: #524  |
| --- |


## Description
Adds welcome confetti.

Reused [confetti.json](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/WordPress/src/main/res/raw/confetti.json) from WP app.

## Testing Instructions
- Create a new account using the onboarding flow, and proceed to the welcome screen.
- Notice that welcome confetti is shown
- Tap "Import podcasts" 
- Notice that Welcome back title is not shown

## Screenshots or Screencast 

Confetti 

https://user-images.githubusercontent.com/1405144/207593996-b783a485-6926-4fc2-851d-8814efecfdab.mp4


## Checklist
- If this is a user-facing change, I have added an entry in CHANGELOG.md
- I have considered whether it makes sense to add tests for my changes
- All strings that need to be localized are in `modules/services/localization/src/main/res/values/strings.xml`
- Any jetpack compose components I added or changed are covered by compose previews
 
#### I have tested any UI changes...

- with different themes
- with a landscape orientation
- with the device set to have a large display and font size
- for accessibility with TalkBack
